mleap-tensorflow/src/test/scala/ml/combust/mleap/tensorflow/converter/MleapConverterSpec.scala

```diff
@@ -0,0 +1,155 @@
+package ml.combust.mleap.tensorflow.converter
+
+import ml.combust.mleap.tensor.{ByteString, Tensor}
+import ml.combust.mleap.core.types.TensorType
+import org.scalatest.FunSpec
+import org.tensorflow.types.TFloat32
+import org.tensorflow.types.TFloat64
+import org.tensorflow.types.TInt32
+import org.tensorflow.types.TInt64
+import org.tensorflow.types.TString
+import org.tensorflow.types.TUint8
+
+class MleapConverterSpec extends FunSpec {
+  def toIndices(dimensions: Traversable[Int]): Seq[Seq[Int]] = {
+    combine(dimensions.map(d => 0 until d))
+  }
+
+  def combine[A](xs: Traversable[Traversable[A]]): Seq[Seq[A]] =
+    xs.foldLeft(Seq(Seq.empty[A])) {
+      (x, y) => for (a <- x.view; b <- y) yield a :+ b
+    }
+
+  def assertSameTensor(lhr: Tensor[_], rhs: Tensor[_]) = {
```

Comment:
Also need to check indices if Sparse

Reply:
Also we needs to test scalar. 
Tensorflow is capable to return scalar tensor shape of [] size of 1 as well.

Reply:
Added Scalar support in commit e33779ef0aeff76eb92fed83fc7e028c7d5ce5ca